From 62c4c2af399dba0566ae69353f127472fa3e3364 Mon Sep 17 00:00:00 2001
From: Ross Lagerwall <rosslagerwall@gmail.com>
Date: Mon, 16 Mar 2015 00:06:07 +0000
Subject: completion: Fix parallel installation

Use install-data-hook rather than install-data-local as
install-data-hook runs after the other install rules have run.  This is
important for a parallel install so that the destination directory
already exists when the symlinks are created.

diff --git a/programs/completion/Makefile.am b/programs/completion/Makefile.am
index 06833e5..971e4ee 100644
--- a/programs/completion/Makefile.am
+++ b/programs/completion/Makefile.am
@@ -35,6 +35,6 @@ uninstall-symlinks:
 		rm -f $(DESTDIR)$(BASHCOMP_DIR)/$$file; \
 	done
 
-install-data-local: install-symlinks
+install-data-hook: install-symlinks
 
 uninstall-local: uninstall-symlinks
-- 
cgit v0.10.2

